src/styles.css

```diff
@@ -1,4 +1,5 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap');
+@import url('https://unpkg.com/leaflet@1.6.0/dist/leaflet.css');
```

Comment:
The same as in `src/index.html`. Please, check how other people do this, for example [here](https://www.geoapify.com/angular-leaflet-step-by-step-tutorial-to-add-a-map/).